run/gem5art/run.py

```diff
@@ -508,44 +508,44 @@ def getRuns(fs_only: bool = False, limit: int = 0) -> Iterable[gem5Run]:
     for run in fsruns:
         yield gem5Run.loadFromDict(run)
 
-def getRunsByTags(tags: str, fs_only: bool = False,
+def getRunsByName(name: str, fs_only: bool = False,
                   limit: int = 0) -> Iterable[gem5Run]:
-    """ Returns a generator of gem5Run objects, which have the field "tags"
-    **exactly** the same as the tags parameter.
+    """ Returns a generator of gem5Run objects, which have the field "name"
+    **exactly** the same as the name parameter.
 
     If fs_only is True, then only full system runs will be returned.
     Limit specifies the maximum number of runs to return.
     """
 
     if not fs_only:
-        seruns = _db.artifacts.find({'type:':'gem5 run', 'tags': tags},
+        seruns = _db.artifacts.find({'type:':'gem5 run', 'name': name},
                                     limit=limit)
         for run in seruns:
             yield gem5Run.loadFromDict(run)
 
-    fsruns = _db.artifacts.find({'type:':'gem5 run fs', 'tags': tags},
+    fsruns = _db.artifacts.find({'type:':'gem5 run fs', 'name': name},
                                 limit=limit)
     for run in fsruns:
         yield gem5Run.loadFromDict(run)
 
-def getRunsByTagsContaining(tags: str, fs_only: bool = False,
+def getRunsByNameContaining(name: str, fs_only: bool = False,
```

Comment:
Let's call this `getRunsByNameLike` since it's similar to a SQL like: https://www.w3schools.com/sql/sql_like.asp

Reply:
This would make this more complicated, but should we care about case sensitivity?

Reply:
changed